code/modules/mob/living/living_defense.dm

```diff
@@ -748,6 +748,8 @@
 			. |= SHOVE_CAN_HIT_SOMETHING
 	if(HAS_TRAIT(src, TRAIT_BRAWLING_KNOCKDOWN_BLOCKED))
 		. |= SHOVE_KNOCKDOWN_BLOCKED
+	if(HAS_TRAIT(src, TRAIT_NO_SIDE_KICK))
+		. &= ~SHOVE_CAN_KICK_SIDE
```

Comment:
This doesn't work
You're flipping no flag, then `carbon/get_shove_flags` is resolved, then it gets added anyways

Add this check there (`carbon/get_shove_flags`)

Reply:
Fixed !!!!